src/AzureClient/Magic/JobsMagic.cs

```diff
@@ -48,7 +49,8 @@ public JobsMagic(IAzureClient azureClient, ILogger<JobsMagic> logger)
                         - A string to filter the list of jobs. Jobs which have an ID, name, or target
                         containing the provided filter parameter will be displayed. If not specified,
                         all recent jobs are displayed.
```

Comment:
If I understand your change below correctly, this line is out of date?
```suggestion
                        the 30 most recent jobs are displayed.
```
It may also be worth using a constant for the default value, similar to the strategy you used with ParameterNameCount.